File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/ErdResult.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

/**
 * @author yudong
 * @date 2023/2/12
 */
@Data
public class ErdResult<T> {
    private Integer code;
    private String msg;
    private T data;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/ErdResult.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/12
 */

@AllOpen
class ErdResult<T> {
    var code: Int? = null
    var msg: String? = null
    var data: T? = null
}
